src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaControllerTest.php

```diff
@@ -888,6 +889,7 @@ public function testPostWithSmallFile()
             'POST',
             '/api/media',
             [
+                'locale' => 'en',
```

Comment:
Is that also mandatory now in the `MediaController`? Then a test case for that is probably missing.